content/tracing/setup/_index.md

```diff
@@ -91,10 +92,48 @@ Find below the list of all available parameters for your `datadog.yaml` configur
 To get a an overview of all the possible settings for APM, take a look at the Agent's [`datadog.example.yaml`][21] configuration file.
 For more information about the Datadog Agent, see the [dedicated doc page][18] or refer to the [`datadog.yaml` templates][19].
 
+### Trace search
+To enable trace search, [services][30] must be flowing into Datadog. Once services are set up, navigate to the [Trace Search & Analytics doc][31] in the Datadog app. Then follow these steps:
```

Comment:
```suggestion
To enable trace search, [services][30] must be flowing into Datadog. Once services are set up, navigate to the [Trace Search & Analytics docs page][31]. Then follow these steps:
```

we try not to use "Datadog app" and just say "Datadog." in this case, since this doc is being linked to, we can just get rid of the whole clause.

also i'd like to replace "doc" with "docs page"